__tests__/scripts.test.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs')
+const inquirer = require('inquirer')
+
+const postinstall = require('../scripts/postinstall').testable
+const preuninstall = require('../scripts/preuninstall').testable
+const helpers = require('../scripts/scriptHelpers')
+
+const TEST_PROFILE_PATH = '/tmp/.npq_test_profile'
```

Comment:
this test will only run on *nix based systems. there's an `os` module which you can use to get a tmpdir and then use `path` module to get the path for OS-specific.

Reply:
@taobojlen I'll merge for now but if you wanted to send a new PR to fix up the tests that would be nice